cohorts/variant_stats.py

```diff
@@ -158,6 +161,45 @@ def maf_somatic_variant_stats(variant, variant_metadata):
         normal_stats = _maf_variant_stats(variant, variant_metadata, prefix="n")
     return SomaticVariantStats(tumor_stats=tumor_stats, normal_stats=normal_stats)
 
+def _vcf_is_strelka(variant_file, variant_metadata):
```

Comment:
Sure :) I usually do search & replace, but this file has a few pre-existing `'` - so will update the few stragglers.